gateway/1.4/modules/ROOT/nav.adoc

```diff
@@ -49,19 +49,19 @@
 ** xref:stub.adoc[Securing and Governing APIs in Local Mode]
 *** xref:flex-local-secure-api-with-basic-auth-policy.adoc[With Basic Authentication and Rate Limiting]
 *** xref:flex-local-secure-api-with-auto-policy.adoc[With an Automated Resource-Level Policy]
-* xref:stub.adoc[Monitoring Flex Gateway]
-** xref:stub.adoc[Monitoring in Connected Mode]
-*** xref:flex-conn-third-party-logs-config.adoc[Configure Logs for Third-Party Services]
+* xref:flex-monitoring.adoc[Monitoring Flex Gateway]
+** xref:flex-conn-monitoring.adoc[Monitoring in Connected Mode]
 *** xref:flex-conn-message-log.adoc[Configure Message Logging]
```

Comment:
Is this file a single task? I ask because we typically use gerunds in the nav. If it's a single task, then the imperative is fine.
(see the multi-task template: https://confluence.internal.salesforce.com/display/MTDT/Use+the+Content+Templates)

Reply:
Same with line 55.

Reply:
I think I wasn't clear. 
The parent task should be gerund, the subtasks should be imperative.

Why is the title imperative in the nav, but gerund in the topic?
See the Style Guide: https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-gerundtitles